doc_use_git.md

```diff
@@ -0,0 +1,42 @@
+# Propel a contribution with GitHub
+
+It is of course possible to contribute directly to the YunoHost documentation, but this is not the most convenient way to do so for both the contributor and the person who will inject your contribution into the documentation. Here is a tutorial to understand and create a contribution to the Yunohost documentation using [Git](https://git-scm.com/) and [github.com](http://github.com/) which is the Git forge service that hosts and stores the YunoHost source code and documentation.
+
+## Create an account on github.com
+To be able to send your contributions via GitHub, you need to have an account on GitHub, to create the account you will need a valid email address that you have access to. GitHub is a powerful tool with many features, the interface can be a little scary at first.
+You don't have to give your first and last names, you can use a nickname (when you register `Username`).
+
+## Fork the YunoHost documentation in your personal repository
+To fork the source code allows you to create a new branch of development of a software source code or in this case the source code of the documentation. By creating a new branch, this allows you to modify the code and add your contributions without altering the code of the `master' branch, which is the public release of the documentation. This allows you not to have to write down everything at once, but to do it in several steps. (Especially for contributions that require more time).
```

Comment:
backtick missing

Reply:
As I understand it, I've added your observations